build/TargetFramework.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+      <CliTargetFramework>netcoreapp3.0</CliTargetFramework>
+      <!-- We only need this until we get a stage0 with a 3.0 SDK. -->
```

Comment:
Should there be a bug tracking removing this? What's the risk? Lack of test coverage from getting the real value of this?